packages/api/src/external/commonwell/patient.ts

```diff
@@ -183,6 +199,16 @@ export async function registerAndLinkPatientInCW(
       getOrgIdExcludeList,
     });
 
+    if (networkLinks) {
```

Comment:
Commonwell is going to be trickier in the sense of making sure the status doesnt get set to completed. Which happens with `storeIdsAndStatus`

Reply:
No longer relevant.